docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -156,3 +139,13 @@ And you're done!
 You may choose to abstract code for `test.before`, `test.beforeEach`, `test.afterEach.always` and `test.after.always` into a separate file.
 
 To see a demo of this configuration file, look at https://github.com/zellwk/ava-mdb-test
+
+## Why `test.serial` instead of `test`
```

Comment:
I fully understand that AVA runs tests in parallel, and your desire to set it up by default. I would love to do that too. 

However, based on @nodkz's explanation, and my personal research over the past month, I strongly believe endpoint testing with Mongoose and AVA is an exception where serial tests would be preferred. 

The setup for parallel tests changes how a person would create their schemas, and thus their app, too much for the additional benefit of running tests in parallel within each file. Besides, if you create multiple AVA test files, each test file still runs in parallel to other files, which shouldn't slow down testing too much. 

I'll be glad to change the docs if you could show me a good way to run endpoint testing in parallel with Mongoose and Supertest.